app/templates/components/job-infrastructure-notification.hbs

```diff
@@ -12,3 +12,11 @@
     {{/if}}
   {{/if}}
 {{/unless}}
+
+{{#if isWindows}}
+  {{#notice-banner}}
+    Windows builds are in early access stage. Please head to the
+    <a href='https://travis-ci.community/c/windows' target='_blank'>Travis CI Community</a>
```

Comment:
You could use the `{{external-link-to}}` here to shorten things up and add the necessary `rel` attrs. See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/ for more details of why we added that component.

Reply:
Wow, I never thought about that! Thanks for sharing the article. And even Same-origin policy seems to [allow `location` changes](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#Window)